invokeai/frontend/web/src/services/api/endpoints/models.ts

```diff
@@ -162,6 +162,8 @@ const buildTransformResponse =
  */
 const buildModelsUrl = (path: string = '') => buildV2Url(`models/${path}`);
 
+// TODO(psyche): Ideally we can share the cache between the `getXYZModels` queries and `getModelConfig` query
```

Comment:
I was trying to figure this out yesterday too

Reply:
We do this now for the gallery, we'll definitely want to implement it before release to avoid spamming network requests